integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -266,9 +276,13 @@ private static String[] optimizeAndTransformFuseMountOptions(AlluxioConfiguratio
     // See https://github.com/libfuse/libfuse/blob/fuse_2_9_3/ChangeLog#L655-L659,
     // and https://github.com/torvalds/linux/commit/78bb6cb9a890d3d50ca3b02fce9223d3e734ab9b.
     // Libfuse3 dropped this option because it's default. Include it doesn't error out.
-    options.add("-obig_writes");
+    String bigWritesOptions = "-obig_writes";
+    options.add(bigWritesOptions);
```

Comment:
I think I made a mistake earlier. Adding big_writes option in libfuse3 will error out. We shouldn't include this option if using libfuse3. Can you help fix?

Reply:
Addressed, PTAL, thanks!